src/keymap.rs

```diff
@@ -394,16 +468,13 @@ impl InputState {
             key = self.emacs_digit_argument(rdr, wrt, digit)?;
         }
         let (n, positive) = self.emacs_num_args(); // consume them in all cases
-        {
-            let bindings = self.custom_bindings.read().unwrap();
-            if let Some(cmd) = bindings.get(&key) {
-                debug!(target: "rustyline", "Custom command: {:?}", cmd);
-                return Ok(if cmd.is_repeatable() {
-                    cmd.redo(Some(n), wrt)
-                } else {
-                    cmd.clone()
-                });
-            }
+
+        if let Some(cmd) = self.custom_binding(wrt, key) {
+            return Ok(if cmd.is_repeatable() {
+                cmd.redo(Some(n), wrt)
```

Comment:
This replaces number returned from the custom action. I.e. I can't return `Move(BackwardChar(10)`.

Reply:
Right. This should not be done for `Action`, only for simple bindings.